src/main/resources/templates/animal/about.ftl

```diff
@@ -0,0 +1,1226 @@
+          href="https://toolbar.qodeinteractive.com/_qode_toolbar/assets/css/qode-demos-modules.css?ver=5.3"
+          type="text/css" media="all">
+    <link rel="stylesheet" id="qode_demos_swiper_css-css"
+          href="https://toolbar.qodeinteractive.com/_qode_toolbar/assets/css/swiper.css?ver=5.3" type="text/css"
+          media="all">
+    <link rel="stylesheet" href="https://bridge82.qodeinteractive.com/wp-content/cache/minify/c138f.css" media="all">
+    <style id="rs-plugin-settings-inline-css" type="text/css">
+        #rs-demo-id {
+        }
+
+    </style>
+    <link rel="stylesheet" href="https://bridge82.qodeinteractive.com/wp-content/cache/minify/2bf8a.css" media="all">
+    <!--[if lt IE 9]>
+    <link rel='stylesheet' id='vc_lte_ie9-css'
+          href='https://bridge82.qodeinteractive.com/wp-content/plugins/js_composer/assets/css/vc_lte_ie9.min.css'
+          type='text/css' media='screen'/>
+    <![endif]-->
+    <script src="https://bridge82.qodeinteractive.com/wp-content/cache/minify/129fc.js"></script>
+    <link rel="https://api.w.org/" href="https://bridge82.qodeinteractive.com/wp-json/">
+    <link rel="EditURI" type="application/rsd+xml" title="RSD"
+          href="https://bridge82.qodeinteractive.com/xmlrpc.php?rsd">
+    <link rel="wlwmanifest" type="application/wlwmanifest+xml"
+          href="https://bridge82.qodeinteractive.com/wp-includes/wlwmanifest.xml">
+    <meta name="generator" content="WordPress 5.3">
+    <link rel="canonical" href="https://bridge82.qodeinteractive.com/">
+    <link rel="shortlink" href="https://bridge82.qodeinteractive.com/">
+    <link rel="alternate" type="application/json+oembed"
+          href="https://bridge82.qodeinteractive.com/wp-json/oembed/1.0/embed?url=https%3A%2F%2Fbridge82.qodeinteractive.com%2F">
+    <link rel="alternate" type="text/xml+oembed"
+          href="https://bridge82.qodeinteractive.com/wp-json/oembed/1.0/embed?url=https%3A%2F%2Fbridge82.qodeinteractive.com%2F&amp;format=xml">
+
+    <script data-cfasync="false" data-pagespeed-no-defer="" type="text/javascript">//<![CDATA[
+        var dataLayer_content = {
+            "pagePostType": "frontpage",
+            "pagePostType2": "single-page",
+            "pagePostAuthor": "admin"
+        };
+        dataLayer.push(dataLayer_content);//]]>
+
+    </script>
+    <script data-cfasync="false">//<![CDATA[
+        (function (w, d, s, l, i) {
+            w[l] = w[l] || [];
+            w[l].push({
+                'gtm.start':
+                    new Date().getTime(), event: 'gtm.js'
+            });
+            var f = d.getElementsByTagName(s)[0],
+                j = d.createElement(s), dl = l != 'dataLayer' ? '&l=' + l : '';
+            j.async = true;
+            j.src =
+                '//www.googletagmanager.com/gtm.' + 'js?id=' + i + dl;
+            f.parentNode.insertBefore(j, f);
+        })(window, document, 'script', 'dataLayer', 'GTM-WCC77WV');//]]>
+
+    </script>
+
```

Comment:
Do we need this js script? What is the purpose of this part?

Reply:
Deleted